fgpyo/util/inspect.py

```diff
@@ -226,7 +228,7 @@ def dict_parse(dict_string: str) -> Dict[Any, Any]:
                 return types.make_enum_parser(type_)
             elif types.is_constructible_from_str(type_):
                 return functools.partial(type_)
-            elif isinstance(type_, type(type(None))):
+            elif type_ is type(None): # type: ignore[E721]
```

Comment:
mypy complains about this check, but using `isinstance` here is incorrect, thus ignoring it.